src/routes/explore/+page.svelte

```diff
@@ -1 +1,227 @@
+                >
+                    {page_title}
+                </button>
+            {/each}
+        </filter_page_tabs>
+    </explore-options>
+
+    <filter-options class="flex items-end gap-3 md:gap-0 justify-between mt-5 md:mt-0">
+        <time-range class="hidden md:flex flex-col md:gap-[0.35vw]">
+            <span class="leading-none text-surface-50 md:text-[1vw]">Time Range</span>
+            <div class="relative flex items-center">
+                <button class="btn absolute right-0 p-0 md:mr-[1vw] md:w-[1.25vw]">
+                    <Chevron class="text-surface-50" />
+                </button>
+                <input
+                    type="text"
+                    placeholder="Any"
+                    class="border-none bg-surface-400 leading-none placeholder:text-surface-50 focus:ring-0 md:w-[11vw] md:rounded-[0.5vw] md:py-[0.8vw] md:pl-[1vw] md:text-[1vw]"
+                    value="All-Time"
+                />
+            </div>
+        </time-range>
+        <genres class="flex flex-col gap-1 md:gap-[0.35vw]">
+            <span class="leading-none text-surface-50 text-base md:text-[1vw]">Genres</span>
+            <div class="relative flex items-center">
+                <button class="btn absolute right-0 p-0 mr-3 md:mr-[1vw] w-5 md:w-[1.25vw]">
+                    <Chevron class="text-surface-50" />
+                </button>
+                <input
+                    type="text"
+                    placeholder="Any"
+                    class="border-none bg-surface-400 leading-none placeholder:text-surface-50 focus:ring-0 md:w-[11vw] w-full md:rounded-[0.5vw] py-3 rounded-lg text-base md:py-[0.8vw] md:pl-[1vw] md:text-[1vw]"
+                />
+            </div>
+        </genres>
+        <year class="hidden md:flex flex-col md:gap-[0.35vw]">
+            <span class="leading-none text-surface-50 md:text-[1vw]">Year</span>
+            <div class="relative flex items-center">
+                <button class="btn absolute right-0 p-0 md:mr-[1vw] md:w-[1.25vw]">
+                    <Chevron class="text-surface-50" />
+                </button>
+                <input
+                    type="text"
+                    placeholder="Any"
+                    class="border-none bg-surface-400 leading-none placeholder:text-surface-50 focus:ring-0 md:w-[11vw] md:rounded-[0.5vw] md:py-[0.8vw] md:pl-[1vw] md:text-[1vw]"
+                />
+            </div>
+        </year>
+        <season class="flex flex-col gap-1 md:gap-[0.35vw]">
+            <span class="leading-none text-surface-50 text-base md:text-[1vw]">Season</span>
+            <div class="relative flex items-center">
+                <button class="btn absolute right-0 p-0 mr-3 md:mr-[1vw] w-5 md:w-[1.25vw]">
+                    <Chevron class="text-surface-50" />
+                </button>
+                <input
+                    type="text"
+                    placeholder="Any"
```

Comment:
This is not the best way to handle things i'm afraid

Reply:
> This is not the best way to handle things i'm afraid

Yes... those are just placeholders
this will be fixed after adding select menu